Authy2FA/App_Start/IdentityConfig.cs

```diff
@@ -55,10 +55,10 @@ public static ApplicationUserManager Create(IdentityFactoryOptions<ApplicationUs
             manager.PasswordValidator = new PasswordValidator
             {
                 RequiredLength = 6,
-                RequireNonLetterOrDigit = true,
+                RequireNonLetterOrDigit = false,

```

Comment:
Ya, think we shouldn't reduce the security in the same just for our own ease of development.  Its too likely that someone will copy/paste the sample direct and not realize that these specific settings were changed.